CHANGELOG.md

```diff
@@ -138,6 +138,10 @@
   [nickgravelyn](https://github.com/nickgravelyn)
   [#757](https://github.com/CocoaPods/Xcodeproj/pull/757)
 
+* Fix crash when adding sub project.
+  [HDB-Li](https://github.com/HDB-Li)
+  [#762](https://github.com/CocoaPods/Xcodeproj/pull/762)
```

Comment:
nit: this should be 851

Reply:
Sorry, modified before I saw these comments.